### PR TITLE
Add bundling of package

### PR DESCRIPTION
**Pull request by andyrichardson**
## Changes
 - Add test to CI
 - Add archiving/bundling script
 - Add deployment of new releases
 - Fix issue of missing assets
 - Bump version to v0.0.5

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -78,18 +99,38 @@ workflows:
           filters:
             tags:
               only: /.*/
+      # - test:
+      #     requires:
+      #       - install
+      #     filters:
+      #       tags:
+      #         only: /.*/
```
Comment (sofiapoh) [inline .circleci/config.yml]: Is there a reason these are commented out?

Comment (andyrichardson) [reply .circleci/config.yml]: @sofiapoh yeah the current snapshots are failing - if you can check / fix the snapshots and push them to this branch, I can add it to CI 👍

Comment (sofiapoh) [reply .circleci/config.yml]: alright!

**Comment by andyrichardson** [comment]
Docker hub is down - will merge later.

**Pull request merged**